File: Assignment_13/Q4.c
////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name    :  Assignment13-Q4.c
//  Descreption :   accept numbers of rows and number of columns from users and display belows pattern.
//  Author      :   Vaishanvi D Shingare
//  Date        :     28/04/2025

// input  : iRow = 4  iCol = 4

/*  output :  4 4 4 4
              3 3 3 3
              2 2 2 2 
              1 1 1 1
*/

////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern( int iRow, int iCol)

{
    int i =1;
    int j =1;
    
    for(i = iRow; i >= 1;i--)
    {
        for(j = 1; j <= iCol;j++)
        {
            printf("%d\t",i);
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0,iValue2= 0;

    printf("Enter number of rows and columns :");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment_47/Q4.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//  File name :     assignment47-4.c
//  Descreption :   write java program which accept number of rows and number of columns from user and display. 
//  Author :        Vaishanvi D Shingare
//  Date :          11/05/2025
//
// Input 3  4   
/*
    *    #   *  #
    *    #   *  #
    *    #   *  #
    *    #   *  #
*/
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                if(j % 2 == 0)
                {
                    System.out.print("#\t");
                }
                else
                {
                    System.out.print("*\t");
                }
            }
            System.out.print("\n");
        }
    }
}

class Q4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}